labs/lab000/lab000.md

```diff
@@ -28,7 +28,7 @@ The clusters for the lab are deployed externally, and the only requisite to foll
 More information about KubeVirt and related components can be found here:
 
 - [KubeVirt](http://kubevirt.io/)
-- [KubeVirt user guide](https://kubevirt.io/user-guide/docs/latest/welcome/index.html)
+- [KubeVirt user guide](https://kubevirt.io/user-guide/welcome/)
```

Comment:
Can you change to simply `https://kubevirt.io/user-guide`? [user-guide/pull/425](https://github.com/kubevirt/user-guide/pull/425) is going to remove the `welcome` uri